src/helpers/ERC721Proof.ts

```diff
@@ -108,7 +109,8 @@ export default class ERC721Proof
     balanceSignature: BalanceSignature,
     eddsaPublicKey: PublicKey
   ) {
-    const [r2, s2] = crypto.getRandomValues(new BigUint64Array(2))
+    const r2 = new Entropy({ charset: '0123456789ABCDEF', bits: 64 }).string()
```

Comment:
Hm, smells like it doesn't work: 
![image](https://user-images.githubusercontent.com/49658988/186114442-51e4e2e8-5065-4f50-bdc9-d79b28c84e4c.png)

Reply:
+ you can extract `'0123456789ABCDEF'` into:
```TS
const charset = '0123456789ABCDEF'
```

Reply:
Oh good idea